Src/AutoFixture.NUnit3/InlineAutoDataAttribute.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework.Interfaces;
+
+namespace Ploeh.AutoFixture.NUnit3
+{
+    /// <summary>
+    /// This attribute acts as a TestCaseAttribute but allow incomplete parameter values, 
+    /// which will be provided by AutoFixture. 
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    [CLSCompliant(false)]
+    public class InlineAutoDataAttribute : AutoDataAttribute
```

Comment:
Just an idea - I can actually reverse the inheritance here, so **AutoDataAttribute** is just a special case of **InlineAutoDataAttribute** that has zero parameters!
Any thoughts @moodmosaic @ploeh